CMakeLists.txt

```diff
@@ -693,7 +693,11 @@ endif()
 add_subdirectory(Externals/imgui)
 add_subdirectory(Externals/implot)
 add_subdirectory(Externals/glslang)
-add_subdirectory(Externals/spirv_cross)
+# SPIRV-Cross is used on Windows for GLSL to HLSL conversion for the Direct3D 11 and Direct3D 12
+# video backends, and on Apple devices for the Metal video backend.
+if((CMAKE_SYSTEM_NAME STREQUAL "Windows") OR APPLE)
```

Comment:
Maybe use `WIN32` as a shorter alternative to `CMAKE_SYSTEM_NAME STREQUAL "Windows"`?

Reply:
Are these defined to be the same thing? If so I'll swap both uses.

Reply:
[The documentation](https://cmake.org/cmake/help/latest/variable/WIN32.html) doesn't say it for sure, but it does say it also applies to 64-bit windows, which is good enough for me.